src/main/java/nl/tudelft/scrumbledore/level/Fruit.java

```diff
@@ -78,4 +82,20 @@ public void setPickable(Boolean bool) {
     pickable = bool;
   }
 
+  /**
+   * Retrieve a set of Sprites to be drawn in the current cycle at the position of this Level
+   * Element.
+   * 
+   * @param steps
+   *          The absolute exact number of steps since the game was started.
+   * @return Sprites to be drawn.
+   */
+  public ArrayList<Sprite> getSprites(double steps) {
+    SpriteStore store = SpriteStore.getInstance();
+    String id = "fruit";
+    ArrayList<Sprite> result = new ArrayList<Sprite>();
+    result.add(store.getAnimated(id).getFrame(posX()));
```

Comment:
"fruit" could be placed directly into .getAnimated(id) to save one line of code, but it is not something that is necessary.
<review